ChangeLog

```diff
@@ -17,6 +17,8 @@ Release date: TBA
 
 * Fix pyreverse diagrams type hinting for classmethods and staticmethods.
 
+* Fix pyreverse diagrams type hinting for methods returning None.
```

Comment:
Could you als add a changelog entry to the `2.13` file? It's in `doc/whatsnew`

I checked and it is also missing for you previous PR. Perhaps you could add that one in this PR as well?

Reply:
The Milestone is currently set to 2.14.0 though (however, the last PR was terminated for 2.14 as well initially and then merged into 2.13.0). 
Should we start including a section for 2.14 in the ``ChangeLog`` already, to avoid confusion?

Reply:
I added it to the 2.13, tell me if you would rather target 2.14 for this change.

Reply:
Let's do `2.13` for this as the review is done and we can merge it :)